app/src/main/java/com/greenfox/peridot/peridot_coz_android/model/api/MockService.java

```diff
@@ -1,22 +1,30 @@
 package com.greenfox.peridot.peridot_coz_android.model.api;
 
 import com.greenfox.peridot.peridot_coz_android.model.pojo.User;
+
+import dagger.Module;
+import dagger.Provides;
 import retrofit2.Call;
 import retrofit2.Callback;
 import retrofit2.Response;
 
-public class MockService implements UserInterface {
+@Module
+public class MockService implements UserInterface{
 
     private static final String TAG = "MockService";
-
     public MockService() {}
 
+    @Provides
```

Comment:
@Phaedra144 move this to a specific `ApiProvider` class which does only providing this